### PR TITLE
tweaking youtube embed, plus some vids ported to youtube

### PR DESCRIPTION
**Pull request by railsagainstignorance**

### CODE REVIEW

File: _includes/youtube_video.html [hunk 1]
```diff
@@ -1 +1 @@
-<iframe class="youtube-iframe" align="center" width="{{ include.width }}" height="{{ include.height }}" src="{{ include.src }}" frameborder="0" allowfullscreen></iframe>
+<iframe class="youtube-iframe" align="center" width="{{ include.width | default: 560 }}" height="{{ include.height | default: 315 }}" src="{{ include.src }}?rel=0" frameborder="0" allowfullscreen></iframe>
```
Comment (Lily2point0) [inline _includes/youtube_video.html]: Video size causes problems on mobile view.

File: _includes/youtube_video.html [hunk 2]
```diff
@@ -1 +1 @@
-<iframe class="youtube-iframe" align="center" width="{{ include.width }}" height="{{ include.height }}" src="{{ include.src }}" frameborder="0" allowfullscreen></iframe>
+<div class="youtube-container"><iframe class="youtube-iframe" align="center" src="{{ include.src }}?rel=0" frameborder="0" allowfullscreen></iframe></div>
```
Comment (seanmtracey) [inline _includes/youtube_video.html:1]: The URL may already have query parameters on it. The `rel=0` may be ignored by some URL parsers in some instance.

Comment (railsagainstignorance) [reply _includes/youtube_video.html:1]: This is a youtube-specific include, so should only be for youtube embed urls.

Comment (seanmtracey) [reply _includes/youtube_video.html:1]: I think we should have a warning somewhere

**Pull request merged**